pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -883,12 +889,15 @@ func TestGarbageCollectorSync(t *testing.T) {
 	}
 
 	sharedInformers := informers.NewSharedInformerFactory(client, 0)
+	metadataInformers := metadatainformer.NewSharedInformerFactory(metadataClient, 0)
 
 	tCtx := ktesting.Init(t)
 	defer tCtx.Cancel("test has completed")
 	alwaysStarted := make(chan struct{})
 	close(alwaysStarted)
-	gc, err := NewGarbageCollector(tCtx, client, metadataClient, rm, map[schema.GroupResource]struct{}{}, sharedInformers, alwaysStarted)
+
+	gc, err := NewGarbageCollector(tCtx, client, metadataClient, rm, map[schema.GroupResource]struct{}{},
+		informerfactory.NewInformerFactory(sharedInformers, metadataInformers), alwaysStarted)
```

Comment:
I don't understand why the informers changed.  If we are unable to use the previous logic, then we broke something.  This is a blocker for merge.

Reply:
reverted to original. Entire file does not have any diff now.